docroot/modules/humsci/hs_field_helpers/src/Plugin/Field/FieldType/DisplayModeField.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Drupal\hs_field_helpers\Plugin\Field\FieldType;
```

Comment:
Why did you choose to create a new field type rather than use one from core or display suite?